examples/flutter_gallery/pubspec.yaml

```diff
@@ -3,13 +3,13 @@ dependencies:
   flutter:
     sdk: flutter
   collection: 1.14.6
-  device_info: 0.2.0
+  device_info: 0.1.2
   intl: 0.15.2
-  connectivity: 0.3.0
+  connectivity: 0.2.1
   string_scanner: 1.0.2
-  url_launcher: 3.0.0
+  url_launcher: 2.0.2
   cupertino_icons: 0.1.1
-  video_player: 0.3.0
+  video_player: 0.2.1
```

Comment:
i suspect you didn't mean to do this

Reply:
No, I **had** to do that.  The gallery on HEAD doesn't build without it.  I'll revert it if someone lands a fix.

Reply:
OK, fixed it.  I had the same problem as aam did...